assets/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+cleanup() {
+  docker stop pulp
+  docker rm pulp
```

Comment:
```suggestion
```
Sorry, I was the one asking for it, but by `rm`ing that container, we loose it's logs. XD